README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`: Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter]: Exposes Prometheus metrics on the cluster status.
```

Comment:
Mixed naming convention `es-master` vs es-client vs Justwatch [Exporter]

My preference would be `es-master`, `es-client` and `es-exporter`

Reply:
Your correct, and I'm correct in fact! The service is already called `es-exporter`, just the documentation isn't correct. I'll update it.